### ISSUE TITLE
UssdCallManager does not try to authenticate Sip connection from UssdGW

### ISSUE THREAD

**Issue by abdulazizali77** [RestComm/Restcomm-Connect#2540]
@FerUy @deruelle @marca56 
`UssdCallManager` allows and processes any Sip INVITE requests without even trying to authenticate request.

**Comment by FerUy** [comment]
@abdulazizali77 that's true. Anyway, let me review 3GPP TS 24.390 spec again, as I'm almost sure the reason we don't do that is that it's not in the spec. In the near future, when mobile phone manufacturers and core networks start supporting USSI (USSD over SIP for the IMS/LTE and beyond), they will surely follow this spec, so we need to stick to it to being able to process their transactions. Will come back with aforementioned review and feedback.
